hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientMapProxy.java

```diff
@@ -489,7 +490,7 @@ public void lock(K key, long leaseTime, TimeUnit timeUnit) {
     public boolean isLocked(K key) {
         checkNotNull(key, NULL_KEY_IS_NOT_ALLOWED);
         final Data keyData = toData(key);
-        ClientMessage request = MapIsLockedParameters.encode(name, keyData, ThreadUtil.getThreadId());
+        ClientMessage request = MapIsLockedParameters.encode(name, keyData, 0);
         ClientMessage response = invoke(request, keyData);
```

Comment:
I removed this parameter in my MultiMap pr. This pr will probably be conflicted with mine. After that I can update and cleanup.